tests/python/unittest/common.py

```diff
@@ -94,6 +95,23 @@ def random_seed(seed=None):
         random.seed(next_seed)
 
 
+def assert_raises_cudnn_disabled(assertion_error=False):
+    def test_helper(orig_test):
+        @make_decorator(orig_test)
+        def test_new(*args, **kwargs):
+            cudnn_disabled = (os.getenv('CUDNN_DISABLED') == "ON")
+            if not cudnn_disabled or mx.context.current_context().device_type == 'cpu':
```

Comment:
Ah that's the environment variable. Could we give it a name that clearly shows that this environment variable is only for test purposes?

Reply:
I have renamed it to `CUDNN_OFF_TEST_ONLY` in the PR: #11470. Let us move the discussion for cudnn off tests to that PR.